guides/common/assembly_logging-and-reporting-problems.adoc

```diff
@@ -1,19 +1,23 @@
 include::modules/con_logging-and-reporting-problems.adoc[]
 
-include::modules/proc_enabling-debug-logging.adoc[leveloffset=+1]
-
 include::modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc[leveloffset=+1]
 
+ifndef::foreman-deb[]
+include::modules/ref_configuring-logging-levels-with-hammer-cli.adoc[leveloffset=+1]
```

Comment:
For context: hammer-cli-admin is a really weird package. Normally Hammer CLI is used to talk to the Foreman REST API, so it can be used remotely. hammer-cli-admin is the very odd exception in that it is completely different: it performs actions on the local system.

On Satellite it is installed by default, but in upstream it isn't. There is no installer support either, so you have to know the package name.

So technically you should have instructions to install it, but I'd really like to discourage people from using it.

Perhaps nothing actionable right now, but providing some context.

Reply:
Nothing actionable here, resolving.